alexa/skill/news.php

```diff
@@ -230,21 +233,23 @@ public function endpoint_content( $args ) {
 				'post_status' => 'publish',
 			) ) );
 
-			$content = '';
+			$content = $card_content = '';
```

Comment:
I am not sure if new WP standards like multiple declarations on a single line.  You might have to break this into two lines.